src/store/importexport/actions.tsx

```diff
@@ -94,12 +94,16 @@ export const exportProject = (
       });
   } else {
     /* ignore export type for now -- online is always ptf */
+    const remProjectId =
+      typeof projectid === 'number'
+        ? projectid.toString()
+        : remoteId('project', projectid, memory.keyMap);
```

Comment:
This should throw an error if we don't have a number, or this should have been caught before now, because we're sending this online, so if we have a guid, we're screwed.

Reply:
Well, I changed all the calls so they pass the guid since all records have guids but not all have remote ids. So if it is a number we make it into a string form that number. If it is a guid, I look it up. and so as you say we should find it. I can add logic to throw an error if this if remProjectId turns out to be undefined but in fact, I think it will throw an error anyway from the API. So I don't know if we actually need to test it as you said we should always have a number. We weren't actually testing it before. We were converting to remoteId as part of the call to exportProject and passing undefined if there was no valid value. (I think the test in line 70 will prevent us from coming to this line if the guid doesn't have a number since we don't show the offline only projects when we have an accessToken